HelpSource/Classes/GridLines.schelp

```diff
@@ -110,6 +110,9 @@ If numTicks is nil: used to guess the ideal numTicks based on the graph size.
 argument:: numTicks
```

Comment:
In the interest of clarifying behavior of `numTicks` and/vs. `tickSpacing`, this description could also be expanded to something like:
"The approximate number of ticks to display, subject to the view bounds and subsequent spacing returned by link::Classes/GridLines#niceNum::."

For reference:
```supercollider
// behavior of GridLines:getParams
(
var dataMin = 0, dataMax =  200;
var plotPixMin = 0, plotPixMax = 1500;
var nticks;

// nticks = 10; // >> returns 11 ticks
// nticks = 11; // >> returns 11 ticks
// nticks = 13; // >> returns 11 ticks
nticks = 15; // >> returns 21 ticks

g = GridLines([dataMin, dataMax].asSpec);
p = g.getParams(
	dataMin, dataMax, plotPixMin, plotPixMax, nticks
).at(\lines).postln;
p.size
)
```
```supercollider
// behavior of GridLines:niceNum
"requested / returned".postln; (0, 2 .. 20).do({|x| postf("% / %\n", x, g.niceNum(x,false)) })
```

Reply:
@mtmccrea would it be okay to leave this for another PR?